src/main/java/ch/njol/skript/bukkitutil/EnchantmentUtils.java

```diff
@@ -40,4 +70,57 @@ public static Enchantment getByKey(String key) {
 		return Enchantment.getByKey(NamespacedKey.minecraft(key));
 	}
 
+	@SuppressWarnings("deprecation")
+	@Nullable
+	public static Enchantment parseEnchantment(String s) {
+		s = s.toLowerCase(Locale.ROOT);
+		// First try to parse from the lang file
+		Enchantment enchantment = PATTERNS.get(s);
+		if (enchantment != null)
+			return enchantment;
+
+		// If that fails, we move forward with getting from key
+		s = s.replace(" ", "_");
+		NamespacedKey key;
+		try {
+			if (s.contains(":"))
+				key = NamespacedKey.fromString(s);
+			else
+				key = NamespacedKey.minecraft(s);
+		} catch (IllegalArgumentException ignore) {
+			return null;
+		}
+		if (key == null)
+			return null;
+		if (HAS_REGISTRY) // Registry added in Bukkit 1.14
+			return Registry.ENCHANTMENT.get(key);
+		return Enchantment.getByKey(key);
+	}
+
+	@SuppressWarnings("null")
+	public static Collection<String> getNames() {
+		return NAMES.values();
+	}
+
+	@SuppressWarnings("null")
+	public static String toString(final Enchantment enchantment) {
+		// If we have a name in the lang file, return that first
+		if (NAMES.containsKey(enchantment))
+			return NAMES.get(enchantment);
+
+		NamespacedKey key = enchantment.getKey();
+		// Else if it's a missing minecraft enchant, just return the key
+		if (key.getNamespace().equalsIgnoreCase("minecraft"))
+			return key.getKey();
```

Comment:
I would personally prefer always including the name space but I'd like opinions from others.

Reply:
I think returning the namespace would be good.

Reply:
your wish is my command 🦖 ... that was supposed to be a thumbs up, but im kind of enjoying the t-rex.
<img width="484" alt="Screenshot 2024-05-13 at 3 21 11 PM" src="https://github.com/SkriptLang/Skript/assets/20008482/9b948d89-90c7-4952-9e70-21425ed8aabe">